### ISSUE TITLE
The source column from the templates should be retained in the final output.

### ISSUE THREAD

**Issue by tbrittoborges** [alevar/ORFanage#2]
Currently, the output GTF CDS entries are annotated as coming from Stringtie. If the source is retained from the template, one can distinguish where it came from.

**Comment by alevar** [comment]
This is an interesting idea. I previously added "orfanage_template" attribute to the transcript features to indicate the source of the conversion, however I do agree a better solution is needed since transcript ID's aren't descriptive enough of the origin file/source.

However, ORFanage may produce an edited version of the source CDS. I feel labeling  the "source" column with the template data could mislead users into thinking the CDS is intact and identical to the source.

To combat both of these problems, I suggest the source column in the GTF is labeled with an impartial "ORFanage" label to indicate it has been added by the software. The Template source, however, is then written into the attributes, alongside other metadata.

I have provided an implementation in the latest version of the source. Note that if "keep_cds" is enabled and a transcript is written with the original CDS, the source column will not be altered.

Again, thank you for your input and interest!
